pulumi/infra/service_shared.py

```diff
@@ -0,0 +1,8 @@
+"""
+Things like consts around cost tuning that should be shared by both
+Lambda and Fargate services.
+"""
+
+
+def get_service_log_retention_days() -> int:
+    return 31
```

Comment:
Might be better to put this into https://github.com/grapl-security/grapl/blob/main/pulumi/infra/config.py